regression/pages/lms/course_page_lms.py

```diff
@@ -22,3 +22,13 @@ def click_resume_button(self):
         Clicks Resume button of the course selected
         """
         self.q(css='.last-accessed-link').first.click()
+
+    def get_pages_name(self):
```

Comment:
Interesting. I think you've put this change up before, but would you mind sending me an example of the results? Will it also give you the sequences within each unit, for example?